encrypted-media-respec.html

```diff
@@ -4051,7 +3834,7 @@
         <h3>User Tracking</h3>
         <section class="informative">
           <h4>Concerns</h4>
-          <p>A third-party host (or any entity, such as an advertiser, capable of getting content distributed to multiple sites) could use a <a def-id="distinctive-identifier"></a> or persistent data, including licenses, keys, key IDs, <a def-id="records-of-key-usage"></a>, or <a def-id="records-of-license-destruction"></a>, stored by or on behalf of the <a def-id="cdm"></a> to track a user across multiple sessions (including across <a def-id="origin">origins</a> and <a def-id="browsing-profile">browsing profiles</a>), building a profile of the user's activities or interests.
```

Comment:
I missed fixing the order consistency here. In the fork, we should swap these for consistency. Alternatively, we can fix this then rebase this PR.